flipperdevices/Flipper-Android-App
# Migrate screenstreaming to new api

**Background**

Migrate screenstreaming to new api

**Changes**

- Migrated screenstreaming module
- Added screenstreaming feature module

**Test plan**

- Open screenstreaming in mobile app
- Close it
- See there's no evet subscription leaks
- See it's working as intended
